src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -121,16 +120,13 @@ <h4 class="modal-title">Benny Charlés's Profile - More Info</h4>
                                     </div>
                                     <div id="visibility-options0" class="panel panel-default" style="display: none;">
                                         <div class="panel-heading">Visibility Options</div>
-                                        <table class="table text-center" style="color:#000;">
+                                        <table class="table text-center" style="color: #000; background: #fff;">
```

Comment:
I see. Try to separate behavioral changes into separate PRs, in case we
need to revert them later.